mozilla-lint-rules/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
 
     testImplementation "com.android.tools.lint:lint:${Versions.android_lint_api}"
     testImplementation "com.android.tools.lint:lint-tests:${Versions.android_lint_api}"
+    testImplementation 'junit:junit:4.12'
```

Comment:
I think we can just do `testImplementation Deps.junitApi` (Just pulled this into https://github.com/mozilla-mobile/fenix/pull/16041/ to make sure the other license tests can run)

Reply:
Great!
I checked out the `Versions` object and it had JUnit 5 so I went for the hardcoded version (definitely expecting a review comment), didn't check the `Deps` object though

I'll update it to `testImplementation Deps.junitApi` then :)